docs/reference/argocd.md

```diff
@@ -1475,6 +1474,12 @@ spec:
       openShiftOAuth: false
       resources: {}
       version: v2.21.0
+      env:
+        - name: ARGO_WORKFLOWS_SSO_CLIENT_SECRET
```

Comment:
This dex example in docs is for a default configurations. Is `ARGO_WORKFLOWS_SSO_CLIENT_SECRET` env set/enabled by default?

Reply:
I didn't see that. Good catch! I'll remove this.